### DISCUSSION TITLE
The is GREAT !!! Found via - APIs for Beginners 2023

### DISCUSSION THREAD

**Discussion by MichaelJWhitmore** [flawiddsouza/Restfox#41] [General]
I really like this software.  Came here from the Free Code Camp video:
https://www.youtube.com/watch?v=WXsD0ZgxjRw
I'm on Windows 11 and not quite sure which install file to use however...

**Comment by flawiddsouza** [comment]
Hi @MichaelJWhitmore, on the [releases](https://github.com/flawiddsouza/Restfox/releases) page, you can use the .exe to install Restfox on Windows:
![image](https://user-images.githubusercontent.com/10164500/226976561-66d1eedb-382c-4afc-90e3-290d2aae3b84.png)

Doesn't matter if you're on WIndows 11 or 10.

Alternatively you can also install Restfox as a chrome app by going to:
https://chrome.google.com/webstore/detail/restfox/fihmegnabgglklkppphibngblkomlhcn

Then you don't need to install anything on the computer.

By the way, I had no idea that Restfox had been featured in a video. So thanks for letting me know. I'm glad more people are finding this.

**Reply by MichaelJWhitmore** [reply]
I looked more closely and figured it out.  Thanks for confirming.  I've moved to the "on the computer" installation and it works great !
